src/js/ScrollArea.jsx

```diff
@@ -370,10 +370,10 @@ export default class ScrollArea extends React.Component {
     }
 
     computeSizes() {
-        let realHeight = this.content.offsetHeight;
-        let containerHeight = this.wrapper.offsetHeight;
-        let realWidth = this.content.offsetWidth;
-        let containerWidth = this.wrapper.offsetWidth;
+        let realHeight = (this.content) ? this.content.offsetHeight : 0;
```

Comment:
I think that brackets around this.content are unnecessary ;)

Reply:
Fixed that too! Sorry for all the delay again 😄